runtime/plugin/lua.vim

```diff
@@ -0,0 +1,5 @@
+lua vim.inspect = require("vim.inspect").inspect
```

Comment:
use just `require("vim.inspect")`

Reply:
No, I've modified the code to make sure only that one function is exported, rather than the full table. So the metatable that has `__call` set isn't there, I tend to avoid them.

Reply:
Exactly what do we gain by not exposing the informative keys of `inspect`? What do we gain by deliberatly breaking the `options.process` argument to the function, which requires some keys? Why can't we just expose the interface offered by `require'inspect'`, and very simply avoid any trouble by breaking the reasonable expectation that `vim.inspect` works the same?

Reply:
Not sure myself, that's what I took away from the gitter convo with you and justinmk.

(e) Ok, I reread and see that in the end justin agreed with you. I'll put back the old require.